classes/controllers/grid/issues/IssueGridHandler.inc.php

```diff
@@ -372,7 +372,9 @@ function clearPubId($args, $request) {
 		import('controllers.tab.pubIds.form.PublicIdentifiersForm');
 		$form = new PublicIdentifiersForm($issue);
 		$form->clearPubId($request->getUserVar('pubIdPlugIn'));
-		return new JSONMessage(true);
+		$json = new JSONMessage(true);
+		$json->setEvent('reloadTab', [['tabsLocator' => '#editIssueTabs', 'tabLocator' => '#identifiersTab']]);
+		return $json;
```

Comment:
Does this change also need to be added to the `clearIssueObjectsPubIds` op?

Reply:
No, I don't think so, since the article DOIs aren't displayed in this modal.